cobra/cmd/license_lgpl.go

```diff
@@ -0,0 +1,188 @@
+package cmd
+
+func initLgpl() {
+	Licenses["lgpl"] = License{
+		Name:            "GNU Lesser General Public License",
+		PossibleMatches: []string{"lgpl", "lesser gpl", "gnu lgpl"},
```

Comment:
A plain LGPL resolves to LGPL 2.0 or later not an LGPL 3.0
And a plain lesser gpl resolves to LGPL 2.1 or later not an LGPL 3.0